### PR TITLE
docs: improve installation instructions

### LINKED ISSUE #60: Improve installation instructions
This project is awesome but it took me more than 2 hours to understand how to build my README file with jsx-readme because the installation instructions aren't clear (at least to me, they are not).

** Would you be willing to help with a PR? **
- [x] Yes, absolutely
- [ ] Yes, with some guidance
- [ ] Unfortunately no time :'-(

## Describe the solution you'd like
The installation instructions should provide all the steps for at least 1 of the ways to build README with ts-node.

## Describe alternatives you've considered
Does not apply

## Additional context
I think installation instructions would be more helpfull if they were like this:

Installation

Add  `jsx-readme` and `ts-node` to your devDependencies
```sh
npm i jsx-readme ts-node -D
```

Add these  configs to your `tsconfig.json`:
```json
{
  "compilerOptions": {
    "resolveJsonModule": true,
    "jsx": "react"
   }
}
```

Create a README.MD template (you may copy the example from this repo `examples/README.md.tsx` and edit to your taste).

Add the following script to your `package.json`:

```json
{
  "scripts": {
    "generate:readme": "ts-node README.md.tsx"
  }
}
```

### PR DESCRIPTION
**Pull request by jruipinto**
Closes #60 

## Checklist:

- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,9 +151,41 @@ import MD, {
...
+
+\`\`\`
+{
+  "scripts": {
+    "generate:readme": "ts-node README.md.tsx"
+  }
+}
+\`\`\`
+
+`;
```
Comment (dbartholomae) [inline README.md:187]: Please inline and use the `jsx-md` components to render the markdown:

```tsx
Add <InlineCode>jsx-readme</InlineCode> and <InlineCode>ts-node</InlineCode> to your <InlineCode>devDependencies</InlineCode>.
<LineBreak />
<CodeBlock language='sh'>
npm i jsx-readme ts-node -D
</CodeBlock>
Add these configs to your <InlineCode>tsconfig.json</InlineCode>:
<LineBreak />
<CodeBlock language='json'>
{
  "compilerOptions": {
    "resolveJsonModule": true,
    "jsx": "react"
   }
}
</CodeBlock>
Create a README.MD template (you may copy the example from this repo examples/README.md.tsx and edit to your taste).
Add the following script to your <InlineCode>package.json</InlineCode>:
<LineBreak />
<CodeBlock language='json'>
{
  "scripts": {
    "generate:readme": "ts-node README.md.tsx"
  }
}
</CodeBlock>
```

**Review by dbartholomae — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jruipinto) [reply README.md:187]: CodeBlock isn´t working as expected. Do you have any suggestion?

imagem

Comment (dbartholomae) [reply README.md:187]: It's due to the `{}` which need to be escaped. You could try the following:

```tsx
<CodeBlock language='json'>
  {`{
  "scripts": {
    "generate:readme": "ts-node README.md.tsx"
  }
}`}
</CodeBlock>
```

**Review by dbartholomae — APPROVED** [review]
Nice! Thanks a lot! :)

**Pull request merged**